src/eventHubManager.ts

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { EventData, EventHubClient, EventPosition, MessagingError, OnError, OnMessage } from "@azure/event-hubs";
+import EventHubManagementClient from "azure-arm-eventhub";
+import * as vscode from "vscode";
+import { Constants } from "./constants";
+import { IoTHubMessageBaseExplorer } from "./iotHubMessageBaseExplorer";
+import { EventHubItem } from "./Model/EventHubItem";
+import { TelemetryClient } from "./telemetryClient";
+import { Utility } from "./utility";
+
+export class EventHubManager extends IoTHubMessageBaseExplorer {
+    private _eventHubClient: EventHubClient;
+
+    constructor(outputChannel: vscode.OutputChannel) {
+        super(outputChannel, "$(primitive-square) Stop Monitoring Custom Event Hub Endpoint", "azure-iot-toolkit.stopMonitorCustomEventHubEndpoint");
+    }
+
+    public async startMonitorCustomEventHubEndpoint(eventHubItem: EventHubItem) {
+        if (this._isMonitoring) {
```

Comment:
build in endpoint and custom endpoint could be moniored at the same time?

Reply:
As we synced offline, yes.